R/standard-data-prep/update-geography-lookups.R

```diff
@@ -0,0 +1,98 @@
+library(readr)
+# @param open_geography_file Lookup file sourced from
+#       https://geoportal.statistics.gov.uk/search?q=LUP_WPC&sort=Date%20Created%7Ccreated%7Cdesc
+#       e.g. data/downloaded_source_data/Ward_to_Westminster_Parliamentary_Constituency_to_Local_Authority_District_to_Upper_Tier_L.csv
+update_pcon_la_lookup <- function(open_geography_file, existing_data_file = "data/la_pcon_hierarchy.csv") {
```

Comment:
Outputs of this function in the console are quite verbose / makes it miss the message you added in there, could  cut out the bits from read_csv() by using `show_col_types = FALSE`?

![image](https://github.com/dfe-analytical-services/dfe-published-data-qa/assets/52536248/7f71c098-1d32-4822-9865-1e0924023974)

Then would consider adding a starting message confirming where it's reading the new file from (e.g. 'Reading X from data/downloaded_source_data', and a closing one to say 'Writing updated data/la_pcon_hierarchy.csv file'?

Reply:
Done